Libraries/oneTBB/tbb-async-sycl/src/tbb-async-sycl.cpp

```diff
@@ -0,0 +1,184 @@
+//==============================================================
+// Copyright © 2019 Intel Corporation
+//
+// SPDX-License-Identifier: MIT
+// =============================================================
+
+#include <cmath>  //for std::ceil
+#include <array>
+#include <atomic>
+#include <iostream>
+#include <thread>
+
+#include <CL/sycl.hpp>
+
+#include <tbb/blocked_range.h>
+#include <tbb/flow_graph.h>
+#include <tbb/global_control.h>
+#include <tbb/parallel_for.h>
+// dpc_common.hpp can be found in the dev-utilities include folder.
+// e.g., $ONEAPI_ROOT/dev-utilities//include/dpc_common.hpp
+#include "dpc_common.hpp"
+
+using namespace cl::sycl;
```

Comment:
Bad coding style.
Even more in a tutorial because a user has no prior idea about what is SYCL namespace or not...

Reply:
Thank you for your input in regards to namespaces. We will get back to you after our internal discussion.”